tests/utils/rpc/request.go

```diff
@@ -93,6 +94,50 @@ func Decode(body []byte, target interface{}) error {
 	return nil
 }
 
+// DecodeScale decodes []body into the target interface.
+// It assumes that the response.Result is a SCALE-encoded value.
+func DecodeScale(body []byte, target interface{}) error {
+	decoder := json.NewDecoder(bytes.NewReader(body))
+	decoder.DisallowUnknownFields()
+
+	var response ServerResponse
+	err := decoder.Decode(&response)
+	if err != nil {
+		return fmt.Errorf("cannot decode response: %s: %w",
+			string(body), err)
+	}
+
+	if response.Version != "2.0" {
+		return fmt.Errorf("%w: %s", ErrResponseVersion, response.Version)
+	}
+
+	if response.Error != nil {
+		return fmt.Errorf("%w: %s (error code %d)",
+			ErrResponseError, response.Error.Message, response.Error.ErrorCode)
+	}
+
+	var result string
+	err = json.Unmarshal(response.Result, &result)
```

Comment:
I believe the intention behind that is to send back the request data to the client. Polkadot-js displays that along with the error message. 

There is no reason to include that in the node response since the client will already have that data. It's better to return just the error message. 

I feel we should just return `ErrResultIsNil`